packages/server-sdk/src/lib/public.types.ts

```diff
@@ -110,12 +119,26 @@ export interface AssociateIdentityParams {
     token: string
 }
 
+export interface GetAuthProviderCallbackParams {
```

Comment:
for consistency can we change here as well since these are not callbacks

Reply:
Done